timescaledb/how-to-guides/hyperfunctions/counter-aggregation.md

```diff
@@ -0,0 +1,15 @@
+# Counter aggregation
+There are two primary types of metrics that can be recorded in a database:
```

Comment:
There are many more types of metrics than this...so I think we need to clarify that these are metrics as used in the sense in application performance monitoring (ie PromQL etc.).

Reply:
Also is this file meant to be here anymore? or is it replaced by the one below?